### DISCUSSION TITLE
Will the joke/concept files still be used?

### DISCUSSION THREAD

**Discussion by piratabrasileiro1** [FunkinCrew/Funkin#2184] [Q&A] [answered]
joke/concept files like lol.png or limo being crushed will still be used in the game or some of them will be removed at some point?

**Comment by piratabrasileiro1** [comment]
By the way, this "ReddGit" is really strange LOL, but at least it's technical

**Comment by Lasercar** [comment, marked as answer]
By what's happened since this:
1. No, they've been removed lol 
2. See first comment (nothing ever happened).

**Comment by piratabrasileiro1** [comment]
ok, thanks for the response
